src/creature/creature.h

```diff
@@ -21,6 +21,7 @@ class Creature
 		std::string creature_type);
 
 	void PrintStats();
+	int GetHP();
```

Comment:
Empty line above. Also, getters and setters should follow the variable naming convention (so it should be written as `get_hp()`)

Reply:
done